src/PlaywrightSharp/Contracts/IWorker.generated.cs

```diff
@@ -0,0 +1,133 @@
+    ///     worker.Close += (_, _) =&gt; Console.WriteLine($"Worker closed {worker.Url}");<br/>
+    /// };<br/>
+    /// <br/>
+    /// Console.WriteLine("Current Workers:");<br/>
+    /// foreach(var pageWorker in Page.Workers)<br/>
+    /// {<br/>
+    ///     Console.WriteLine($"\tWorker: {pageWorker.Url}");<br/>
+    /// }
+    /// </code>
+    /// </summary>
+    public partial interface IWorker
+	{
+		/// <summary>
+		/// <para>
+		/// Emitted when this dedicated <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API)">WebWorker</a>
+		/// is terminated.
+		/// </para>
+		/// </summary>
+		event EventHandler Close;
+	
+		/// <summary>
+		/// <para>Returns the return value of <paramref name="expression"/>.</para>
+		/// <para>
+		/// If the function passed to the <see cref="IWorker.EvaluateAsync"/> returns a [Promise],
+		/// then <see cref="IWorker.EvaluateAsync"/> would wait for the promise to resolve and
+		/// return its value.
+		/// </para>
+		/// <para>
+		/// If the function passed to the <see cref="IWorker.EvaluateAsync"/> returns a non-[Serializable]
+		/// value, then <see cref="IWorker.EvaluateAsync"/> returns <c>undefined</c>. Playwright
+		/// also supports transferring some additional values that are not serializable by <c>JSON</c>:
+		/// <c>-0</c>, <c>NaN</c>, <c>Infinity</c>, <c>-Infinity</c>.
+		/// </para>
+		/// </summary>
+		/// <param name="expression">
+		/// JavaScript expression to be evaluated in the browser context. If it looks like a
+		/// function declaration, it is interpreted as a function. Otherwise, evaluated as an
+		/// expression.
+		/// </param>
+		/// <param name="arg">Optional argument to pass to <paramref name="expression"/>.</param>
+		Task<T> EvaluateAsync<T>(string expression, object arg = null);
+	
+		/// <summary>
+		/// <para>Returns the return value of <paramref name="expression"/> as a <see cref="IJSHandle"/>.</para>
+		/// <para>
+		/// The only difference between <see cref="IWorker.EvaluateAsync"/> and <see cref="IWorker.EvaluateHandleAsync"/>
+		/// is that <see cref="IWorker.EvaluateHandleAsync"/> returns <see cref="IJSHandle"/>.
+		/// </para>
+		/// <para>
+		/// If the function passed to the <see cref="IWorker.EvaluateHandleAsync"/> returns
+		/// a [Promise], then <see cref="IWorker.EvaluateHandleAsync"/> would wait for the promise
+		/// to resolve and return its value.
+		/// </para>
+		/// </summary>
+		/// <param name="expression">
+		/// JavaScript expression to be evaluated in the browser context. If it looks like a
+		/// function declaration, it is interpreted as a function. Otherwise, evaluated as an
+		/// expression.
+		/// </param>
+		/// <param name="arg">Optional argument to pass to <paramref name="expression"/>.</param>
+		Task<IJSHandle> EvaluateHandleAsync(string expression, object arg = null);
+	
+		string Url { get; }
+
+
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
```

Comment:
Is this a bug in the generator?

Reply:
No, it's due to the fact we're migrating one by one. The reference *will* be there, when we move all the interfaces over.